chapters/functions.tex

```diff
@@ -2010,6 +2010,7 @@ \subsubsection{Simple Types}\label{simple-types}
 An exception is made when the argument is of the form \lstinline!size($\ldots$, $\ldots$)!. In this case the corresponding C type is \lstinline!size_t!.
 
 Strings are \textsc{nul}-terminated (i.e., terminated by \lstinline[language=C]!'\0'!) to facilitate calling of C functions.
+Strings may contain Unicode and for 8-bit char in C they are coded as UTF-8.
```

Comment:
I think I would rather want to combine the 2 sentences into one.

Reply:
Done now.
I stated that it only applies for C-implementations with 8-bit characters (as on practically all modern computers); I don't see the need to consider other cases yet.

Reply:
@sjoelund, could you please resolve if you are satisfied with the new sentence.